These icons are referred to by the QT3-based frontend, but not
included in the project. SPEC file copies the from QT4 frontend
directory.

Lubomir Rintel <lkundrak@v3.sk>

diff -up VirtualBox-2.1.0/src/VBox/Frontends/VirtualBox/VBoxUI.pro.icons VirtualBox-2.1.0/src/VBox/Frontends/VirtualBox/VBoxUI.pro
--- VirtualBox-2.1.0/src/VBox/Frontends/VirtualBox/VBoxUI.pro.icons	2008-12-30 02:06:39.000000000 +0100
+++ VirtualBox-2.1.0/src/VBox/Frontends/VirtualBox/VBoxUI.pro	2008-12-30 02:08:51.000000000 +0100
@@ -62,40 +62,66 @@ IMAGES	= images/tpixel.png \
 	images/vm_show_logs_disabled_32px.png \
 	images/vm_start_32px.png \
 	images/vm_start_disabled_32px.png \
+	images/os_archlinux_64.png \
 	images/os_archlinux.png \
+	images/os_debian_64.png \
 	images/os_debian.png \
 	images/os_dos.png \
 	images/os_ecs.png \
+	images/os_fedora_64.png \
 	images/os_fedoracore.png \
+	images/os_fedora.png \
+	images/os_freebsd_64.png \
 	images/os_freebsd.png \
+	images/os_gentoo_64.png \
 	images/os_gentoo.png \
 	images/os_l4.png \
 	images/os_linux22.png \
+	images/os_linux24_64.png \
 	images/os_linux24.png \
+	images/os_linux26_64.png \
 	images/os_linux26.png \
+	images/os_linux_other.png \
+	images/os_mandriva_64.png \
 	images/os_mandriva.png \
+	images/os_netbsd_64.png \
 	images/os_netbsd.png \
 	images/os_netware.png \
+	images/os_openbsd_64.png \
 	images/os_openbsd.png \
+	images/os_opensolaris_64.png \
 	images/os_opensolaris.png \
+	images/os_opensuse_64.png \
 	images/os_opensuse.png \
+	images/os_os2ecs.png \
+	images/os_os2_other.png \
 	images/os_os2warp3.png \
-	images/os_os2warp4.png \
 	images/os_os2warp45.png \
+	images/os_os2warp4.png \
+	images/os_other.png \
+	images/os_redhat_64.png \
 	images/os_redhat.png \
+	images/os_solaris_64.png \
 	images/os_solaris.png \
+	images/os_ubuntu_64.png \
 	images/os_ubuntu.png \
 	images/os_unknown.png \
-	images/os_win2k.png \
+	images/os_win2k3_64.png \
 	images/os_win2k3.png \
+	images/os_win2k8_64.png \
 	images/os_win2k8.png \
+	images/os_win2k.png \
 	images/os_win31.png \
 	images/os_win95.png \
 	images/os_win98.png \
 	images/os_winme.png \
 	images/os_winnt4.png \
+	images/os_win_other.png \
+	images/os_winvista_64.png \
 	images/os_winvista.png \
+	images/os_winxp_64.png \
 	images/os_winxp.png \
+	images/os_xandros_64.png \
 	images/os_xandros.png \
 	images/state_aborted_16px.png \
 	images/state_discarding_16px.png \
